tools/clang/test/CodeGenSPIRV/method.consume-structured-buffer.consume.hlsl

```diff
@@ -44,8 +44,9 @@ float4 main() : A {
 // CHECK-NEXT: [[prev:%\d+]] = OpAtomicISub %int [[counter]] %uint_1 %uint_0 %int_1
 // CHECK-NEXT: [[index:%\d+]] = OpISub %int [[prev]] %int_1
 // CHECK-NEXT: [[buffer3:%\d+]] = OpAccessChain %_ptr_Uniform_T %buffer3 %uint_0 [[index]]
-// CHECK-NEXT: [[ac:%\d+]] = OpAccessChain %_ptr_Uniform_v3float [[buffer3]] %int_0 %int_3 %int_1
-// CHECK-NEXT: [[val:%\d+]] = OpLoad %v3float [[ac]]
+// CHECK-NEXT: {{%\d+}} = OpLoad %T [[buffer3]]
+
+// CHECK:      [[val:%\d+]] = OpLoad %v3float {{%\d+}}
```

Comment:
is the `{{%\d+}}` here the same id as the `{{%\d+}}` on the previous line? If yes, use `[[name...:%\d+]]` for the previous line and this line. It is important to check what is being loaded here.

Also, if possible use `CHECK-NEXT` here instead of `CHECK`. If this instruction is right after the previous, we should be able to use CHECK-NEXT. If not, I'd like to know what instructions are in between the previous instruction and this one.